dist/server/carrot-server-clt.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+
+START_HOME=$(dirname "$(realpath "$0")")
+echo Carrot server home directory is "${START_HOME}"
+
+cd "${START_HOME}" || exit
+
+. ./setdist.sh
+webapps="libs/${RELEASE}-${APP_PORT}"
+rm -rf "${webapps}"
+mkdir "${webapps}"
+cd "${webapps}" || exit 1
+tar zxvf "../${DISTRIBUTION}" >/dev/null
+cd ../..
+for ix in $(find "${webapps}"); do
+  CPATH=${ix}\:${CPATH}
+done
+
+export JVM_OPTS="-cp .:${CPATH} ${APP_OPTS}"
+
+#===== find pid =====
+# shellcheck disable=SC2120
+pid() {
+  echo "$(ps -aef | grep "${INSTANCE_NAME}" | grep -v grep | awk {'print $2'})"
+}
+
+#===== start carrot =====
+start() {
+  PID=$(pid)
+  if [ ! -z "${PID}" ]; then
+    echo Server and port number in use PID="${PID}"
+    exit 1
+  fi
+
+  exec_cmd="${JAVA_HOME}/bin/java ${JVM_OPTS} org.bigbase.carrot.redis.CarrotMain ${APPS_PARAMS}"
+  echo "${exec_cmd}"
+  nohup ${exec_cmd} >>logs/catalina.log &
+  echo "Carrot instance ${INSTANCE_NAME} is staring on PID ${PID}, please wait..."
+
+  sleep 2
+  PID=$(pid)
+
+  if [ ! -z "${PID}" ]; then
+    echo "Carrot instance ${INSTANCE_NAME} successfully started. PID ${PID}"
+    # TODO
+    # status
+    exit 0
+  fi
+
+  echo "Carrot instance failed to start on instance name ${INSTANCE_NAME}"
+  exit 1
+}
+
+#==== stop carrot ====
+stop() {
+  if_continue=$1
+
+  PID=$(pid)
+  if [ ! -z "${PID}" ]; then
+    kill -4 "${PID}"
+    sleep 3
+    echo "carrot server, instance ${INSTANCE_NAME}. PID ${PID} successfully stopped"
+    ##TODO
+    #pelase add grace full stop here...
+  fi
+
+  PID=$(pid)
+  if [ ! -z "${PID}" ]; then
+    echo "Carrot server still running on instance ${INSTANCE_NAME} and can't be stopped for some reason. PID ${PID}"
+  else
+    echo No instances of carrot server are runnning.
+  fi
+
+  if [ -z "${if_continue}" ]; then
+    exit 0
+  fi
+}
+
+#===== reboot =====
+reboot() {
+  stop 1
+  sleep 3
+  start
+}
+
+#===== status =====
+status() {
+  echo
+  echo "$(wget ${HEALTH_MONITOR})"
+  echo
+
+  exit 0
+}
+
+#===== usage =====
+usage() {
+  echo
+  echo Usage:
+  # shellcheck disable=SC2102
+  echo \$\> \./carrot-server-clt.sh [start]\|[stop]\|[reboot]\|[status]
```

Comment:
please remove status

Reply:
removed for now but betted to create ping available from wget to make sure server started successfully 
Will create ticket.